src/config/env/stg.json

```
@@ -0,0 +1,33 @@
+{
+  "CHAIN_ID": "1",
+  "LOCAL_STORAGE_KEY": "builder",
+  "EMAIL_SERVER_URL": "https://decentraland.org/subscribe",
+  "PEER_URL": "https://peer-lb.decentraland.org",
```

We could change it to peer:
```suggestion
  "PEER_URL": "https://peer.decentraland.org",
```